README.md

```diff
@@ -3,13 +3,13 @@
 copyleft-next is a new, post-post-modern, non-weak copyleft license
 inspired by, though different from, the GNU GPL.
 
-Contributions of patches, ideas, and criticism are welcome (see the file
-CONTRIBUTING.md for more information).  In particular, the Copyleft-Next
-Project actively seeks the participation of open source/free software
-project developers who may not have considered helping with license
-drafting before. License drafting is fundamentally about establishing
-policies for projects, and therefore input from developers is typically
-more valuable than input from lawyers.
+Contributions of patches, ideas, and criticism are welcome (see the
+file CONTRIBUTING.md for more information).  In particular, the
+Copyleft-Next Project actively seeks the participation of open
+source/free software project developers who may not have considered
+helping with license drafting before. License drafting is
+fundamentally about establishing policies for projects, and therefore
+input from developers of the highest value.
```

Comment:
Shouldn't this be "is of the highest value."? Sentence reads a bit weird to me now.